other/Map tools/Hoenn.tmx

```diff
@@ -59,12 +59,7 @@
  </layer>
  <layer id="9" name="Dungeons" width="100" height="60">
   <data encoding="base64" compression="zstd">
-   KLUv/WDAXJ0OAKaQMzQAzQgUvDqpx0JJvvVop3appzXKhWWmeoIEj8Ef+P//B94NZab////6H4i4fInwIPHrDYEHIgAoACUA34LrcL7NyNfxiXwhIB+fh2kjsJU20uPY+DRGoi3UPp6tOiYlG0mHs2m2DYcNX4dPQ8bAVuA0QMAW4MJngVTho3ChD/RB+B5cxSBXtSYkpi6gzcHWIMELwauTioCYeqiW5nXglbGWTWVj2VZSULaTGV4YXieMjKmLKQvTq/QiYSouEUYv0SuEmaKYsEnYAGwKPi/PZ6jwDAgWSQGwAgQhGFYBEoAQ6GIEEUWQEAwR/FcEVxRNQLMy8pVUPVUErreKpZSoXuY5fpLIq+1W1ls/gCJwcvKz7h9aI+Bjvrj0rfPnhk6Sjn/ES2scsMjJSfjk5HigbUnyLB56AifBiLegS72eRR5NSsRh1J8H9W7IeV+hMjBI6PzKVPx2A/lPhYNKK5/qDH2sbpwuHFQe+RbTi8wcbFGhIAR8l4QRkJNUI0CTKB0VOYk8zEkg6PmrXTYvy68AwNPlIXDXhQ7tNVQrFNzB9warAVtxH6Y+a0nRoAZswX2Y+MTCgIbiJ3EKhTPZcWHDJMZtAF8UTMJH2o/su/H0WvUD
-  </data>
- </layer>
- <layer id="19" name="Dungeon Extension" width="100" height="60">
-  <data encoding="base64" compression="zstd">
-   KLUv/WDAXKUAACgAABAGEgQAYQZwAO7uMHx0eYAF
+   KLUv/WDAXB0NAMLOKCdQd0UBH5GI3AUgZiEhZgHP5HBW+F/lDLZLROBkASEAAKAEI1iumA/38rl8LafynT5TSekjfaOTy+SiuDxCH+j7iC6RK+Q6Po7P87nNt/FpauKCuCSuiAeHq4Fh4VLwdK7NxZGJ2WY2GI3GldFyYXFVsGwrppQNxUabaCPZRuwDsn2Yp7NxEraBDWE7MCjYCFQujOt0mZpAwAZQIl2jFQJdnwNmqCALCBVnAeCCi0FhFRJAEChSBBFG8BDBHxFAEIGCURRNwLFh8ZVsdRWO11vVaJx0FZMqGXf8JLWRNV6ITaSBkvuUN/BU0kvwst46CvhT1gg+0C9yvHXSZLaske3icnhySJRtif0sctHkWJTL2c8SdPiBeTcd+5qhPjAJOD8zE48a4FVaOKjk8ymPhT4vrzgXDioJ+RbzPsgc7KHQZwDfJejdkJOJLgeafGnWyEk6L5wMDHo++oaal1EWJOBpd0fA9HNhU43rkbI4QNS1BmzefZj2I5WD8xqwMfdhys8pDOggfjIXXJxJLd80TF63gfgwY1LPo/kBesqevqV+
```

Comment:
You've made changes to the Hoenn map file, but haven't updated the image.

Reply:
Gimme a bit, dunno why did I change it, probably some water stuffs

Reply:
Removed